packages/core/src/2d/atlas/SpriteAtlas.ts

```diff
@@ -0,0 +1,53 @@
+import { RefObject } from "../../asset/RefObject";
+import { Engine } from "../../Engine";
+import { Sprite } from "../sprite/Sprite";
+
+/**
+ * Sprite Atlas.
+ */
+export class SpriteAtlas extends RefObject {
+  /** @internal */
+  _spritesMap: Record<string, Sprite> = {};
```

Comment:
You store sprites in a map, If two `Sprite`s have the same name, you will lose one.

Reply:
@GuoLei1990 , I know this is the case, but if a developer gets multiple `Sprites` when calling `getSprite`, he probably doesn’t know which one to use, so I think the work of avoiding the same name should be given to the developer.

Reply:
@GuoLei1990 , In addition, I will check the same name when packaging the atlas. Do you think it is feasible, I know your solution, use an `Array` or `Record<string, Sprite[]>`